vulnerabilities/tests/test_data_dump.py

```diff
@@ -85,3 +90,30 @@ def test_ubuntu_data_dump(self):
         reference = VulnerabilityReference.objects.filter(reference_id='CVE-2002-2439')[0]
         self.assertEqual(reference.reference_id, 'CVE-2002-2439')
         self.assertTrue(Package.objects.filter(name='gcc-4.6')[0].name, 'gcc-4.6')
+
+    def test_archlinux_data_dump(self):
```

Comment:
@lohani2280 can you also create a test that effectively asserts the results of our discussions in https://github.com/nexB/vulnerablecode/pull/33#issuecomment-473662522  and https://github.com/nexB/vulnerablecode/pull/33#issuecomment-473662522  ?

Reply:
@pombredanne Can you please elaborate a bit more on what kind of test you want to look for?. Based on the comment in https://github.com/nexB/vulnerablecode/pull/33#issuecomment-473662522,  I think I have included the corresponding tests and not getting what else have I missed.

Reply:
@lohani2280 You are doing limited assertions based on object counts and spot checks. You are not doing a test of all the data that were inserted. So I could change your code behavior quite a bit and still pass the tests. The purpose of testing is also to cast in stone your requirements and design such that you can then update the code and be confident that you have a tests safety net.

So IMHO you should do also an assertion on the whole data set serialized from the DB (possibly using some of API code for that)

And a common way is to store the expected data on disk in the repo. You can check some of these in ScanCode packagedcode tests for instance